io.github.denzilferreira.asteroid_shooter.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "io.github.denzilferreira.asteroid_shooter",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "asteroid-shooter",
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
+    "--device=all",
+    "--socket=pulseaudio",
+    "--share=ipc"
+  ],
+  "modules": [
+    {
+      "name": "python3-pygame",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"pygame>=2.0\" --no-build-isolation"
+      ],
+      "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/49/cc/08bba60f00541f62aaa252ce0cfbd60aebd04616c0b9574f755b583e45ae/pygame-2.6.1.tar.gz",
+            "sha256": "56fb02ead529cee00d415c3e007f75e0780c655909aaa8e8bf616ee09c9feb1f"
+        }
+      ]
+    },
+    {
+      "name": "asteroid-shooter",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D app.sh /app/bin/asteroid-shooter",
+        "install -Dm644 spaceshooter.py /app/spaceshooter.py",
+        "install -Dm644 asteroid.png /app/asteroid.png",
+        "install -Dm644 player1.png /app/player1.png",
+        "install -Dm644 player2.png /app/player2.png",
+        "install -Dm644 laser.wav /app/laser.wav",
+        "install -Dm644 explosion.wav /app/explosion.wav",
+        "install -Dm644 powerup.wav /app/powerup.wav",
+        "install -Dm644 space_shooter_loop.wav /app/space_shooter_loop.wav",
+        "install -Dm644 requirements.txt /app/requirements.txt",
+        "install -d /app/demo",
+        "install -Dm644 demo/demo.gif /app/demo/demo.gif",
+        "install -Dm644 demo/demo-v2.gif /app/demo/demo-v2.gif",
+        "install -D launcher/io.github.denzilferreira.asteroid_shooter.png /app/share/icons/hicolor/512x512/apps/io.github.denzilferreira.asteroid_shooter.png",
+        "install -D launcher/io.github.denzilferreira.asteroid_shooter.desktop /app/share/applications/io.github.denzilferreira.asteroid_shooter.desktop",
+        "install -D io.github.denzilferreira.asteroid_shooter.metainfo.xml /app/share/metainfo/io.github.denzilferreira.asteroid_shooter.metainfo.xml"
+      ],
+      "sources": [
+        {
+          "type": "dir",
```

Comment:
never use source `type: dir`

Reply:
first time submitting to FlatHub :blush: The source is on my repo, so now I see this needs to be type git :+1: I'll update this.